prover/crates/bin/prover_autoscaler/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "zksync_prover_autoscaler"
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[dependencies]
+zksync_core_leftovers.workspace = true
+zksync_vlog.workspace = true
+zksync_utils.workspace = true
+zksync_types.workspace = true
+zksync_config = { workspace = true, features = ["observability_ext"] }
+zksync_prover_job_monitor.workspace = true
+zksync_protobuf_config.workspace = true
+
+anyhow.workspace = true
+async-trait.workspace = true
+axum = "0.7.5"
```

Comment:
It's better to add such dependencies into workspace deps, and then use `axum.workspace = true`(To prevent possible version mismatch in the future)

Reply:
Done